docs/security/host-certs.md

```diff
@@ -87,14 +87,55 @@ If your institution is not in the list and Let's Encrypt certificates do not mee
 
 1. Install the [host](#installing-host-certificates) or [service](#installing-service-certificates) certificate
 
-Requesting Let's Encrypt Certificates
--------------------------------------
+Requesting Host Certificates Using [Let's Encrypt](https://letsencrypt.org/)
+----------------------------------------------------------------------------
+
+[Let's Encrypt](https://letsencrypt.org/) is a free, automated, and open CA frequently used for web services;
+see the [security team's position on Let's Encrypt](https://opensciencegrid.github.io/security/LetsEncryptOSGCAbundle/)
+for more details.  Let's Encrypt can be used to obtain host certificates as an alternative
+to InCommon if you do not have an InCommon subscription.
+
+The `letsencrypt` software (AKA `certbot`) can be obtained from the EPEL 7 yum repo:
+
+``` console
+root@host # yum install certbot
+```
+
+If you have any service running on port 80, you will have to disable it temporarily to obtain
+certificates, as the letsencrypt needs to bind on it temporarily in order to verify the host.
+For instance, if you already have an HTCondor-CE set up and running, stop the
+[CE View service](https://opensciencegrid.github.io/docs/compute-element/install-htcondor-ce/#install-and-run-the-htcondor-ce-view)
+(which listens on port 80).
```

Comment:
What if they don't have a CE View service running? Maybe say "...if you already have an HTCondor-CE running with HTCondor-CE View..."

Reply:
Yeah that makes sense, fixing.